x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/middleware.test.ts

```diff
@@ -23,6 +23,11 @@ import { endpointListReducer } from './reducer';
 import { endpointMiddlewareFactory } from './middleware';
 import { getEndpointListPath } from '../../../common/routing';
 
+jest.mock('../../policy/store/policy_list/services/ingest', () => ({
```

Comment:
I was noticing that in the `middleware` tests, we were hitting errors as the test ran.  I mocked these so that to code flow wasn't interrupted with those and we got a more realistic pass through the code

Reply:
Ahh - Ok. Yeah, I been meaning to circle back around on those. It may be safe to move this to the `setEndpointListApiMockImplementation()`, but we can revisit it at another time.